environments/typescript/recommended.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   },
 
   plugins: [
-    'typescript',
+    '@typescript-eslint',
```

Comment:
Please add this as a dependency of this project. I think it is missing by accident; all the other plugins which are used ie. for React environment *are* bundled with this ruleset so TS plugin should also be bundled. 🙏 Thanks!

Reply:
Done 👍